# The desired frequency for BSM generation
bsm_generation_frequency: 10.0

#Enable/disable rotation of BSM ID during vehicle operations
bsm_id_rotation_enabled: true